### DISCUSSION TITLE
how to enable webGL2?

### DISCUSSION THREAD

**Discussion by mostend** [wailsapp/wails#1496] [Q&A]
how to enable webGL2? now is working on webGL.

**Comment by leaanthony** [comment]
The version of WebGL is whatever is supported by the native webviews. We cannot control that.
We will however try and keep using the latest WebKit libraries for latest features.

**Reply by mostend** [reply]
can we add startup parameters are like starting Chrome?

**Reply by stffabi** [reply]
On what platform are you testing it?

I've successfully tested WebGL2 support with latest Wails on
- Windows 11 Version 10.0.22000.740 - WebView2 Runtime 103.0.1264.37
- MacOS 12.4 - M1

On Linux it seems that at least Fedora and Ubuntu compile their WebKit2 packages without WebGL2 support. So in those cases we can't support it. Do you know a distribution where they activate WebGL2 support in the WebKit2 package?
